### PR TITLE
Strip invalid tcp_connect fields in the web_connectivity test helper

### PR DESCRIPTION
**Pull request by hellais**
This is the ooni-backend side fix of:
https://github.com/TheTorProject/ooni-probe/issues/727

### CODE REVIEW

File: oonib/tests/test_bouncer.py [hunk 1]
```diff
@@ -72,6 +72,22 @@
     test-helper: {fake_test_helper: 'fake_hostname'}
 """
 
+production_bouncer_file = """
+collector:
+  httpo://ihiderha53f36lsd.onion:
+    collector-alternate:
+    - {address: 'https://a.collector.ooni.io:4441', type: https}
```
Comment (bassosimone) [inline oonib/tests/test_bouncer.py]: `https://b.collector.ooni.io`?

File: oonib/tests/test_bouncer.py [hunk 2]
```diff
@@ -72,6 +72,22 @@
...
+    - {address: 'https://das0y2z2ribx3.cloudfront.net', front: a0.awsstatic.com, type: cloudfront}
+    test-helper: {dns: '213.138.109.232:57004', http-return-json-headers: 'http://38.107.216.10:80',
+      ssl: 'https://213.138.109.232', tcp-echo: 213.138.109.232, traceroute: 213.138.109.232,
+      web-connectivity: 'httpo://7jne2rpg5lsaqs6b.onion'}
+    test-helper-alternate:
+      web-connectivity:
+      - {address: 'https://a.web-connectivity.th.ooni.io:4442', type: https}
```
Comment (bassosimone) [inline oonib/tests/test_bouncer.py]: IIRC `https://b.web-connectivity.th.ooni.io`?

File: oonib/tests/test_bouncer.py (hunk 1 above)
Comment (hellais) [inline oonib/tests/test_bouncer.py]: Actually the canonical bouncer is currently using this address. Though anyways what is in here doesn't really matter for the purpose of the unittest.

File: oonib/tests/test_bouncer.py (hunk 2 above)
Comment (hellais) [inline oonib/tests/test_bouncer.py]: Same thing as above. The canonical bouncer is still advertising the old address. We should switch it though when we update to the version in here.

File: oonib/tests/test_bouncer.py (hunk 1 above)
Comment (bassosimone) [inline oonib/tests/test_bouncer.py]: (I don't think it's relevant, however)

File: oonib/tests/test_bouncer.py (hunk 2 above)
Comment (bassosimone) [inline oonib/tests/test_bouncer.py]: (ditto)

**Pull request merged**